alpine/docker-entrypoint.sh

```diff
@@ -26,6 +26,8 @@ if [[ "$1" == "kong" ]]; then
       setcap cap_net_raw=+ep /usr/local/openresty/nginx/sbin/nginx
     fi
 
+    setcap CAP_NET_BIND_SERVICE=+ep /usr/local/openresty/nginx/sbin/nginx
```

Comment:
This breaks the above `setcap` command: it does not *add* a capability, but *set* the capabilities of the program.
You'll need to collect all the capabilities wanted together and do it in one setcap call.